src/studio-header/DesktopHeader.jsx

```diff
@@ -14,9 +14,22 @@ import messages from './Header.messages';
 // Assets
 import { CaretIcon } from './Icons';
 
+// Data
+import StudioApiService from '../data/services/StudioApiService';
+
 class DesktopHeader extends React.Component {
   constructor(props) { // eslint-disable-line no-useless-constructor
     super(props);
+    this.state = {
```

Comment:
What about the MobileHeader? Do you think we should include the lockup in the mobile view?

Reply:
I think I had assumed that because the course_id hadn't been included in the MobileHeader, I wouldn't update the lockup for the mobile view. I think it is worth adding though, so I will update the code.